.circleci/config.yml

```diff
@@ -49,7 +49,18 @@ jobs:
       - restore-go-mod-cache
       - run:
           name: lint and test
-          command: make test-circle
+          command: make test-circle && make test-coverage
+      - run:
+          name: Upload Coverage Results to Codacy
+          command: |
+              if [ -x "$(which curl)" ]; then
+                  curl -Ls https://coverage.codacy.com/get.sh > get.sh
+              elif [ -x "$(which wget)" ] ; then
+                  wget -qO - https://coverage.codacy.com/get.sh > get.sh
+              else
+                  printf "Could not find curl or wget, please install one."
+              fi
+              source get.sh report --force-coverage-parser go -r coverage.out
```

Comment:
I had to create script, because we cannot force go parser in the orb. 
FYI: https://circleci.com/developer/orbs/orb/codacy/coverage-reporter
https://docs.codacy.com/coverage-reporter/uploading-coverage-in-advanced-scenarios/#golang